.github/workflows/phpunit.yml

```diff
@@ -16,7 +16,6 @@ jobs:
           - "lowest"
           - "highest"
           - "locked"
-          - "development"
```

Comment:
Dropped on purpose: we already build against unstable `php`, we don't need to follow every dependencies' dev step

Reply:
At least not anymore thanks to `renovatebot`